lib/src/package_config.dart

```diff
@@ -19,11 +19,6 @@ class PackageConfig {
   /// Packages configured.
   List<PackageConfigEntry> packages;
 
-  /// Date-time the `.dart_tool/package_config.json` file was generated.
-  ///
-  /// `null` if not given.
-  DateTime? generated;
```

Comment:
I would consider not removing the option entirely, just change the default to not write an entry if not given a `DateTime` instead of using `DateTime.now()` as fallback ...